topics.py

```diff
@@ -38,6 +40,7 @@ class Camera(Topic):
     set_overlay = 'camera/set_overlay'
     push_overlays = 'camera/push_overlays'
     errors = '/v3/camera/errors'
+    online_root = '/v3/camera/video-feed/status'
```

Comment:
Can you change the key of this topic to something more suitable? Instead of `online_root` I think it might be better if it was like `video_feed_online` or something similar. I would rather have `online` reserved for knowing if the RPI is currently on/off.

Reply:
Yup! Actually did this before but forgot to push :/
1d320cb